src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml

```
@@ -27,6 +27,7 @@ imports:
 
 parameters:
     sylius_core.public_dir: "%kernel.project_dir%/web"
+    sylius_core.taxation.default_strategy: true
```

```suggestion
    sylius_core.taxation.shipping_address_based_taxation: true
```

Perhaps we could also consider putting to configuration tree and restricting values of this parameter to strings only